validator/rpc/intercepter.go

```diff
@@ -40,8 +40,7 @@ func (s *Server) JwtHttpInterceptor(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		// if it's not initialize or has a web prefix
 		if !strings.Contains(r.URL.Path, api.WebUrlPrefix+"initialize") && // ignore some routes
-			!strings.Contains(r.URL.Path, api.WebUrlPrefix+"health/logs") &&
-			strings.Contains(r.URL.Path, api.WebUrlPrefix) {
+			!strings.Contains(r.URL.Path, api.WebUrlPrefix+"health/logs") {
```

Comment:
Do you need the prefix? Could you not just check:
```
if !strings.Contains(r.URL.Path, "initialize") && // ignore some routes
			!strings.Contains(r.URL.Path, "health/logs") {
```
Is there more than one expected endpoint? If not we could just check that one?

Reply:
Sorry looks like my browser was displaying things weirdly, this comment is irrelevant

Reply:
yeah i think it'll be better with the prefix to indicate it, I'm afraid of other health endpoints down the line if we change that.